app/js/components/pages/Job.jsx

```diff
@@ -45,7 +45,7 @@ const Job = ({ plugin, action, inputs, metadata, submitJob, cancelJob, children
                 </fieldset>
             )}
 
-            { action.parameters.map(({ name, type, ast }) =>
+            { action.parameters.map(({ name, type, ast, default: def }) =>
```

Comment:
`default` is a JS keyword, so using it broke linting. this `{ default: def }` is just inline renaming it to a non-keyword `def`